ingestion/src/metadata/ingestion/source/database/postgres/queries.py

```diff
@@ -296,3 +296,22 @@
     FROM
         aggregate_hierarchy;
 """
+
+POSTGRES_GET_STORED_PROCEDURES = """
+    SELECT proname AS procedure_name,
+        nspname AS schema_name,
+        proargtypes AS argument_types,
+        prorettype::regtype AS return_type,
+        prosrc AS definition
+    FROM pg_proc
+    JOIN pg_namespace ON pg_proc.pronamespace = pg_namespace.oid
+    WHERE prokind = 'p';
+"""
+
+POSTGRES_GET_STORED_PROCEDURES_QUERIES = """
+    SELECT 
+        total_exec_time,
+        query
+    FROM pg_stat_statements
+    WHERE query LIKE 'CALL%';
+"""
```

Comment:
we do not want the query which calls sp, we will not be able to extract queries that were executed during sp execution so lets just add a comment for that and skip this query call.